docs/source/cli_help/config.rst

```diff
@@ -0,0 +1,82 @@
+.. _config:
+
+config
+-----
+Documentation about available ``git machete`` config keys and environment variables that change the command's default behavior.
+
+Note: ``config`` is not a command as such, just a help topic (there is no ``git machete config`` command).
+
+**Config keys:**
+    * ``machete.github.{{remote,organization,repository}}``:
+
+        When executing ``git machete github <subcommand>`` command, GitHub API server URL will be inferred from ``git remote``.
+        You can override this by setting the following git config keys:
+            Remote name
+                E.g. ``machete.github.remote`` = ``origin``
+
+            Organization name
+                E.g. ``machete.github.organization`` = ``VirtusLab``
+
+            Repository name
+                E.g. ``machete.github.repository`` = ``git-machete``
+
+        To do this, run ``git config --local --edit`` and add the following section:
+
+        .. code-block:: ini
+
+            [machete "github"]
+                organization = <organization_name>
+                repository = <repo_name>
+                remote = <remote_name>
+
+    * ``machete.overrideForkPoint.<branch>.{{to,whileDescendantOf}}``
+
+        Executing ``git machete fork-point --override-to=<revision> [<branch>]`` sets up a fork point override for <branch>.
+        The override data is stored under ``machete.overrideForkPoint.<branch>.to`` and ``machete.overrideForkPoint.<branch>.whileDescendantOf`` git config keys.
+
+    * ``machete.status.extraSpaceBeforeBranchName``
+
+        To make it easier to select branch name from the ``status`` output on certain terminals
```

Comment:
Pls state the default (`false` IIRC) explicitly in the first sentence, as you did for `machete.worktree.useTopLevelMacheteFile`

Reply:
(I mean, it is implied, but still)